Common/app/ota/ota_update_task.c

```diff
@@ -174,7 +181,7 @@
 /**
  * @brief Task priority of OTA agent.
  */
-#define otaexampleAGENT_TASK_PRIORITY            ( tskIDLE_PRIORITY + 1 )
+#define otaexampleAGENT_TASK_PRIORITY            ( tskIDLE_PRIORITY + 3 )
```

Comment:
```suggestion
```

Reply:
Will do this in a follow up PR after fixing the task priorities of all tasks.